ui/apps/platform/src/utils/searchUtils.ts

```diff
@@ -185,10 +185,13 @@ export function flattenFilterValue<UndefinedFallback>(
 export function getListQueryParams(
     searchFilter: SearchFilter,
     sortOption: ApiSortOption,
-    page: number,
-    pageSize: number
+    page?: number,
```

Comment:
Most backend APIs can support un-paginated requests, so we don't need to enforce that these values are set.

In this specific use case, we need to request all attached collections by id without pagination.